### ISSUE TITLE
Capacitor 6.0

### ISSUE THREAD

**Issue by TC-jw** [appcues/appcues-capacitor-plugin#154]
Hi there, is there any plan for supporting the latest version Capacitor?

**Comment by andretortolano** [comment]
Thank you for bringing this up.

we can start investigating what updating to Capacitor 6 will entail, is this currently a blocker for your project?

**Comment by TC-jw** [comment]
yes, unable to upgrade capacitor 6 and can't use some of plugins because it has a dependency

**Comment by mmaatttt** [comment]
Hi TC-jw, we've just release version 4.3.2 which supports Capacitor 6.
